app/src/main/kotlin/co/netguru/baby/monitor/client/common/YesNoDialog.kt

```diff
@@ -0,0 +1,70 @@
+package co.netguru.baby.monitor.client.common
+
+import android.content.DialogInterface
+import android.os.Bundle
+import android.support.annotation.StringRes
+import android.support.v4.app.DialogFragment
+import android.support.v7.app.AlertDialog
+import co.netguru.baby.monitor.client.R
+import co.netguru.baby.monitor.client.common.extensions.requireArguments
+import org.jetbrains.anko.bundleOf
+
+class YesNoDialog : DialogFragment() {
+
+    companion object {
+        private const val KEY_TITLE = "key:title"
+        private const val KEY_MESSAGE = "key:message"
+        private const val KEY_POSITIVE = "key:positive"
+        private const val KEY_REQUEST_CODE = "key:request_code"
+
+        fun newInstance(
+            @StringRes title: Int,
+            message: String,
+            @StringRes positiveButton: Int = R.string.dialog_yes,
+            requestCode: Int = -1,
+            params: Bundle? = null
+        ) = YesNoDialog().apply {
+            arguments = bundleOf(
+                KEY_TITLE to title,
+                KEY_MESSAGE to message,
+                KEY_POSITIVE to positiveButton,
+                KEY_REQUEST_CODE to requestCode
+            ).also {
+                if (params != null) {
+                    it.putAll(params)
+                }
+            }
+        }
+    }
```

Comment:
https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout

Accordingly to the Kotlin Coding Conventions companion object should be at the bottom of the class